indico/modules/events/contributions/templates/management/contrib_report_filter.html

```diff
@@ -0,0 +1,106 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+
+{% macro _render_column_selector(item, filter_choices) %}
+    {% set filters = filters['items'] %}
+    <div class="i-label report-column {%- if filter_choices %} filter{% endif %}"
+         data-id="{{ item.personal_data_type.name if item.personal_data_type else item.id }}">
+        <a class="title {% if filter_choices %}icon-filter{% endif %}" data-toggle="dropdown">
+            {{- item.title -}}
+        </a>
+        {% if filter_choices %}
+            <ul class="dropdown" style="display: none">
+                {% for value, caption in filter_choices.iteritems() %}
+                    <li>
+                        <input type="checkbox" name="field_{{ item.id }}" value="{{ value }}"
+                               id="field_{{ item.id }}_option_{{ value }}"
+                               {% if value in filters[item.id] %}
+                                   checked
+                               {% endif %}>
+                        <label for="field_{{ item.id }}_option_{{ value }}">
+                            {{ caption }}
+                        </label>
+                    </li>
+                {% endfor %}
+            </ul>
+        {% endif %}
+    </div>
+{% endmacro %}
+
+<div class="action-box">
+    <div class="section">
+        <div class="icon icon-wrench"></div>
+        <div class="toolbar right">
+            <a class="i-button js-reset-btn icon-close warning" type="button">
+                {%- trans %}Clear filters{% endtrans -%}
+            </a>
+        </div>
+        <div class="text">
+            <div class="label">Customize contribution list</div>
+            <div>
+                {% set filter_icon = '<i class="icon-filter"></i>' | safe %}
+                {%- trans -%}
+                    Change the configuration of the contribution list by applying filtering ({{ filter_icon }}) options.
+                {%- endtrans -%}
+            </div>
+        </div>
+    </div>
+</div>
+<div class="success-message-box js-clear-filters-message" style="display: none">
+    <div class="message-text">
+        {%- trans -%}
+            All filters have been cleared.
+        {%- endtrans -%}
+    </div>
+</div>
+<form class="report-filter" method="POST">
+    <h3>{% trans %}Available filtering options{% endtrans %}</h3>
+
+    {% for item in filterable_items.itervalues() %}
+        {% set filter_choices = item.get('filter_choices') %}
```

Comment:
assigning it to a separate variable seems overkill. also, you can most likely use `item.filter_choices` as long as `_render_column_selector` works fine with Jinja's `Undefined` object